src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -142,7 +143,16 @@ private void sendExternalEmail(Objection objection, CompanyProfileApi companyPro
             logMap.put(LOG_OBJECTION_ID_KEY, objection.getId());
             apiLogger.errorContext(httpRequestId, "Error sending customer email", e, logMap);
 
+            updateObjectionStatus(objection, httpRequestId, ObjectionStatus.ERROR_EXTERNAL_EMAIL);
             throw e;
         }
     }
+
+    private void updateObjectionStatus(Objection objection, String requestId, ObjectionStatus newStatus) {
+        objection.setStatus(newStatus);
+        objection.setHttpRequestId(requestId);
+        objection.setCreatedOn(LocalDateTime.now());
```

Comment:
createdOn is set only when the objection is created, we should have a new field for the status timestamp - maybe statusChangedOn? this records when the status changed.